tests/__snapshots__/UnifiedSchema.test.ts.snap

```diff
@@ -2971,7 +2971,7 @@ type WorkflowRun implements EntityInterface & Node {
   _id: GlobalID!
   cachedResults: String
   collectionId: Int!
-  createdAt: DateTime!
```

Comment:
Do you know why my DateTime's are always trying to change? @jfoo1984

Reply:
I don't have any idea, but I did some searching and found a couple links that could help, though I don't think they're our exact issue.  I'm not sure exactly where this type is coming from, but the timestamps in our sample JSONs are ISO8601 dates, so that's probably why it was changing it to that type?  I'm not really sure how to fix this from the links below though. 


https://github.com/github-community-projects/graphql-client/issues/17
https://the-guild.dev/graphql/scalars/docs/scalars/date-time

Reply:
is there a problem if you use `ISO8601DateTime`?

Reply:
might also be worth checking with The Guild team.

Reply:
Yeah I also noticed the snapshot test seems to be changing the types back and forth nondeterministically? (not completely sure though, didn't dig)